wrf.rb

```diff
@@ -28,6 +28,8 @@ def install
       (buildpath/"WPS").install resource("WPS")
   
       wrf_args = std_cmake_args + %w[
+        -DCMAKE_C_COMPILER=gcc
```

Comment:
This won't do what you want it to. Homebrew doesn't ship GCC without a suffix since it conflicts with Apples `gcc` which is an alias of their clang.

If you're going to go this route I would do:
```suggestion
        -DCMAKE_C_COMPILER=#{FORMULA["gcc"].opt_bin}/gcc-9
```

You'll need to change the previous line to end with `%W[` (capital w to get string interpolation).

But this will be a little bit fragile (it will break in the next major GCC release). You could also query GCC's major version number, but I don't think that it's worth it. It works with clang, AFAICT. You can mint a new release, or point the formula at a particular git SHA if you want to make sure people don't stub their toes trying to use the formula without `--HEAD`.

My recommendation is to remove this, and just use the latest version of wrf-cmake in the formula until you have your next release.

Reply:
Sorry for the confusion here but I was aware that `gcc` was an alias for apple clang. My understanding was that this was the preferred way to build in Homebrew. In this case don't you think we should keep the `gcc` (alias) as it default to apple clang but prevents cmake to build with ifort or whatever else is the default Fortran compiler?

Reply:
Homebrew is very careful about sanitizing the build environment (with the possible exception of a Fortran compiler installed in a particular location). There are hundreds (at least) of CMake formula in homebrew, and they all work as expected by letting CMake find Homebrew's C compiler stubs. The logic around the C compiler is much more robust and battle tested and I would be quite surprised if Homebrew is finding another C compiler.

Concerning Fortran, see my comment above.

Reply:
Sorry, that wasn't as clear as it could have been. The point that I was trying to communicate, is that the way it the formula is currently setup is the *canonical* way to specify CMake formula (no explicit telling CMake which C compiler to use).

Let's treat Fortran in the other comment; I'm curious which OS you're on and where ifort is installed.